src/main/resources/langugage_single-ru.properties

```diff
@@ -0,0 +1,113 @@
+add=Добавлять
+adding.new.password=Добавление нового пароля
+appName=Сохранить мой пароль
```

Comment:
Please, do not change the initial appName, as provided in the guide for translators